src/turrets/Turret.cpp

```diff
@@ -94,7 +95,54 @@ void Turret::pickUpTurret() {
     texture.setAlpha(0);
 }
 
-// checks if there are any enemies in the turret's coverage area, this is not yet defined
+/**
+ * Checks if there are any enemies in the turret's coverage area.
+ * Jamie, March 1
+ * Revised by Rob, March 5
+ */
 bool Turret::targetScanTurret() {
+    //Get map of all zombies
+    const auto& mapZombies = GameManager::instance()->getZombies();
+
+    unsigned int closestZombieId = 0;
+    float closestZombieDist = std::numeric_limits<float>::max();
+
+    // Detect zombies
+    bool detect = false;
+    for (const auto& item : mapZombies) {
+        const auto& zombie = item.second;
+        const float zombieX = zombie.getX();
+        const float zombieY = zombie.getY();
+
+        const float xDelta = abs((abs(zombieX - ZOMBIE_WIDTH / 2) - abs(getX() - TURRET_WIDTH / 2)));
+        const float yDelta = abs((abs(zombieY - ZOMBIE_HEIGHT / 2) - abs(getY() - TURRET_HEIGHT / 2)));
+        const float distance = sqrt(pow(xDelta, 2) + pow(yDelta, 2));
```

Comment:
Don't use `pow` method for this
It's used for arbitrary exponentiation
For simple squaring, doing `xDelta * xDelta` is faster